src/main/java/org/example/services/AuthService.java

```diff
@@ -55,6 +56,14 @@ public String login(final LoginRequest loginRequest)
         }
         return generateJwtToken(validateUser);
     }
+    public void checkEmailExists(final String email) throws SQLException,
+            DoesNotExistException {
+        LoginRequest loginRequest = new LoginRequest(email, "");
```

Comment:
As the getUser() only uses the email address maybe this should only pass in the email address

Reply:
This can't, because the LoginRequest object uses JsonParams to assign variables, and so having another constructor causes really awkward issues. Plus designing a new method using only email would be overkill.

Reply:
You could change the function in the Dao to just accept an email as this is all this function requires and therefore do not need to change the constructor.